src/components/modals/style/modal.scss

```diff
@@ -0,0 +1,18 @@
+.modal-body {
+  border-radius: 15px !important;
+  padding: 2rem !important;
+  text-align: center !important;
+}
+
+@media (min-width: 768px) {
+  .modal-body {
+    width: 400px !important;        // Limit the width of the modal and
+    margin-left: -200px !important; // to center the Modal
```

Comment:
How does this look on a phone or tablet?

Reply:
![galaxys5](https://user-images.githubusercontent.com/35084243/34979464-9f815266-faa1-11e7-843a-65619a237390.PNG)
![ipad](https://user-images.githubusercontent.com/35084243/34979469-a1c6e48c-faa1-11e7-8557-0638c51ecaaa.PNG)
![ipadpro](https://user-images.githubusercontent.com/35084243/34979472-a38a8f1c-faa1-11e7-8e0a-38cf9ab93d2e.PNG)

I think it looks reasonably well.